pkg/kubectl/cmd/set/set_env.go

```diff
@@ -123,8 +123,7 @@ type EnvOptions struct {
 
 	Cmd *cobra.Command
 
-	UpdatePodSpecForObject func(obj runtime.Object, fn func(*api.PodSpec) error) (bool, error)
```

Comment:
should update `f.UpdatePodSpecForObject()`to `o.UpdatePodSpecForObject()` better

Reply:
OK